blockstack/blockstack-android
# Add Network and account API

This PR
* adds access to getNameInfo method, and all other methods listed in blockstack.js for network and transactions.
* adds network propery to BlockstackSession for the current network config
* updates blockstack.js to commit https://github.com/blockstack/blockstack.js/commit/211455e10b86d298e58e607367d7f4dd2583e709
* adds a button in example to get name info of dev_android_sdk.id.blockstack
* adds tests for getNameInfo

To test
1. checkout branch 'feature/name_info`
1. start example in Android Studio
1. see new button at very bottom
